app/assets/javascripts/application.js

```diff
@@ -16,7 +16,20 @@
 //= require bootstrap
 //= require bootstrap-sprockets
 //= require turbolinks
-//= require growlyflash
 //= require moment
```

Comment:
That shouldn't have been pushed, it can be removed. I was trying something out and ended up not using it.